packages/build/src/npm-packages/bump.spec.ts

```diff
@@ -19,9 +16,14 @@ describe('npm-packages bump', function () {
     'mongosh-version.ts'
   );
 
+  let testBumper: PackageBumper;
+
   beforeEach(function () {
     fsWriteFile = sinon.stub(fs, 'writeFile');
     fsWriteFile.resolves();
+
+    testBumper = new PackageBumper();
+    sinon.stub(testBumper, 'spawnSync').resolves();
```

Comment:
Can we turn these 'stored as properties with the expectation to mock them later' methods into regular properties that get passed through the constructor?

The previous dependency injection code was primarily annoying imo because functions would take more and more arguments, but the general principle of explicitly passing in a dependency is still a good one